wildfly-jakartaee-ear-archetype/src/main/resources/archetype-resources/integration-tests/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>${groupId}</groupId>
+        <artifactId>${rootArtifactId}</artifactId>
+        <version>${version}</version>
+    </parent>
+    <artifactId>${rootArtifactId}-integration-tests</artifactId>
+    <packaging>jar</packaging>
+    <name>${rootArtifactId} - integration-tests</name>
+    <description>This module contains the arquillian integration tests for the enterprise application.</description>
+
+    <dependencies>
+
+        <!-- Import the CDI API, we use provided scope as the API is included in WildFly / JBoss EAP -->
```

Comment:
Why do you mention JBoss EAP?

Reply:
I copied the text from the other profiles. "JBoss EAP" is there since the first version of the archetype (JBoss 7) ;-). I can remove them all if you consider this reasonable.